requirements.txt

```diff
@@ -7,4 +7,5 @@ python-etcd
 python-dateutil
 gevent>=1.0
 greenlet>=0.3.2
+gstatus
```

Comment:
We need to build gstatus on our own and I have added dependency already here.
@TimothyAsirJeyasing plz do the needful.